src/include/miopen/datatype.hpp

```diff
@@ -150,15 +152,14 @@ inline KernelBuildParameters GetDataTypeKBP(miopenDataType_t type)
     case miopenHalf: use_fp16 = 1; break;
     case miopenFloat: use_fp32 = 1; break;
     case miopenInt8: use_int8 = 1; break;
-    case miopenInt8x4: use_int8x4 = 1; break;
     case miopenBFloat16: use_bfp16 = 1; break;
     case miopenInt32: use_int32 = 1; break;
     case miopenDouble: use_fp64 = 1; break;
     case miopenFloat8: use_fp8 = 1; break;
     case miopenBFloat8: use_bfp8 = 1; break;
+    case miopenInt8x4: // fallthrough
     default:
-        MIOPEN_THROW(
-            "Only float, half, bfloat16, int8, int8x4, float8, bfloat8 data type is supported.");
+        MIOPEN_THROW("Only float, half, bfloat16, int8, float8, bfloat8 data types are supported.");
```

Comment:
Int32 and double are missing

Reply:
Resolved in https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2452/commits/9ac65ea622cc28aa2f674a2ec5d39b8f14c21a7d